MANIFEST.in

```diff
@@ -1,29 +1,17 @@
 include README.rst
 include CHANGES.rst
 
-include ez_setup.py
-include ah_bootstrap.py
-include setup_helpers.py
-include setup.cfg
-
-recursive-include *.pyx *.c *.pxd
-recursive-include docs *
 recursive-include licenses *
-recursive-include cextern *
-recursive-include scripts *
 recursive-include asdf-standard/schemas *.yaml
```

Comment:
The items in `asdf-standard` should already be included because this is in `package_data` specified in `setup()`, right?

Reply:
No, it turns out not.  Copying `package_data` must be a "build" step.  When I opened up an sdist I discovered that the schemas directory doesn't exist in asdf/ yet.

Reply:
Ah, ok.  Interesting.  I guess since the `wheel` is built, that's how it gets included in the `bdist_wheel`.

Reply:
Now that these are listed as package_data, we no longer need them in the manifest.